src/query/storage/converter_test.go

```diff
@@ -93,9 +93,13 @@ func TestFailingExpandSeriesValidPools(t *testing.T) {
 	validTagGenerator := func() ident.TagIterator {
 		return seriesiter.GenerateSingleSampleTagIterator(ctrl, testTags)
 	}
-	iters := seriesiter.NewMockSeriesIterSlice(ctrl, validTagGenerator, 4, 2)
-	invalidIters := make([]encoding.SeriesIterator, 2)
-	for i := 0; i < 2; i++ {
+
+	iters := seriesiter.NewMockSeriesIterSlice(ctrl, validTagGenerator, 8, 2)
+	// Add 3 failing iterators, despite the pool having size 2; there can be
+	// slight timing inconsistencies which can sometimes cause.
```

Comment:
nit: `which can sometimes cause ...` looks like it's missing the end of the sentence.